web/tests/message_list_view.test.js

```diff
@@ -786,44 +786,69 @@ test("render_windows", ({mock_template}) => {
         assert.equal(view._render_win_end, end);
     }
 
+    function verify_move_and_no_move_range(move_target, opts = {}) {
```

Comment:
It would probably be better to do a refactor to using this function as a separate commit before the main commit. I can do that next time I'm working if that's preferred before merging.

Reply:
Yes definitely better to refactor and then change the behavior; way way more readable.